### PR TITLE
Add stubs for types.get_original_bases

### PR DESCRIPTION
**Pull request by Gobot1234**
ref: https://github.com/python/cpython#101827

### CODE REVIEW

File: stdlib/types.pyi [hunk 1]
```diff
@@ -563,6 +566,9 @@ def prepare_class(
     name: str, bases: tuple[type, ...] = (), kwds: dict[str, Any] | None = None
 ) -> tuple[type, dict[str, Any], dict[str, Any]]: ...
 
+if sys.version_info >= (3, 12):
+    def get_orignal_bases(__cls: Any) -> tuple[Any, ...]: ...
```
Comment (JelleZijlstra) [inline stdlib/types.pyi:570]:
```suggestion
    def get_original_bases(__cls: Any) -> tuple[Any, ...]: ...
```

Comment (Gobot1234) [reply stdlib/types.pyi]: Oops, thanks

File: stdlib/types.pyi [hunk 2]
```diff
@@ -563,6 +566,9 @@ def prepare_class(
     name: str, bases: tuple[type, ...] = (), kwds: dict[str, Any] | None = None
 ) -> tuple[type, dict[str, Any], dict[str, Any]]: ...
 
+if sys.version_info >= (3, 12):
+    def get_original_bases(__cls: Any) -> tuple[Any, ...]: ...
```
Comment (AlexWaygood) [inline stdlib/types.pyi:570]: It needs to be passed a type, right?

```suggestion
    def get_original_bases(__cls: type) -> tuple[Any, ...]: ...
```

Comment (Gobot1234) [reply stdlib/types.pyi:570]: I'm really not having a good night lol, yeah you're right

Comment (Gobot1234) [reply stdlib/types.pyi]: Actually should that be type[Any] or does it not matter?

Comment (AlexWaygood) [reply stdlib/types.pyi]: It shouldn't matter

Comment (Gobot1234) [reply stdlib/types.pyi]: Oh yeah prepare_class doesn't subscript further

**Review by AlexWaygood — APPROVED** [review]
Thanks!

**Pull request merged**